frontend/cypress/e2e/celltypes.cy.js

```diff
@@ -50,7 +50,7 @@ describe('Cell Type Editing', () => {
         .trigger('mouseover', canvasWidth / 4, canvasHeight / 4)
         .click()
         .wait(100);
-      cy.contains('Untitled 1 (1)');
+      // cy.contains('Untitled 1 (1)');
```

Comment:
Consider adding a quick comment about why this is commented out (e.g. if we plan to uncomment this at some future date when the cellType canvas supports overlaps in a performant way). Otherwise we could consider removing outright.

NBD either way!